### PR TITLE
docs(scanv4): Add new example values to secured-cluster chart

### PR DESCRIPTION
**Pull request by mclasmeier**
## Description

Adds example value files for the secured-cluster-services Helm chart for Scanner V4.

## Checklist
- [x] ~~Investigated and inspected CI test results~~
- [x] ~~Unit test and regression tests added~~
- [x] ~~Evaluated and added CHANGELOG entry if required~~
- [X] ~~Determined and documented upgrade steps~~
- [X] Documented user facing changes

If any of these don't apply, please comment below.

### Here I tell how I validated my change

TODO(replace-me)  
Use this space to explain **how you validated** that **your change functions exactly how you expect it**.
Feel free to attach JSON snippets, curl commands, screenshots, etc. Apply a simple benchmark: would the information you
provided convince any reviewer or any external reader that you did enough to validate your change.

It is acceptable to assume trust and keep this section light, e.g. as a bullet-point list.

It is acceptable to skip testing in cases when CI is sufficient, or it's a markdown or code comment change only.  
It is also acceptable to skip testing for changes that are too taxing to test before merging. In such case you are
responsible for the change after it gets merged which includes reverting, fixing, etc. Make sure you validate the change
ASAP after it gets merged or explain in PR when the validation will be performed.  
Explain here why you skipped testing in case you did so.

Have you created automated tests for your change? Explain here which validation activities you did manually and why so.

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: image/templates/helm/stackrox-secured-cluster/values-public-scanner-v4.yaml.example [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+#     #
+#     # For performance reasons it is HIGHLY RECOMMENDED to use a PVC as persistence backend
+#     # for Scanner V4 DB.
+#     #
+#     # If a default storage class exists on the cluster, then the persistence configuration
+#     # defaults to a PVC for a fresh installation. On a cluster without a default storage class the
+#     # storage class to be used can be configured explicitly (see below).
+#     #
+#     # On a cluster without a default storage class the persistence backend defaults to
+#     # an emptyDir (persistence.none=true).
```
Comment (johannes94) [inline image/templates/helm/stackrox-secured-cluster/values-public-scanner-v4.yaml.example:179]: This is a suggestion based on the Operator UI comment for this config

```suggestion
#     # Configures how Scanner V4 stores its persistent data.
#     # You can use a persistent volume claim (the recommended default), a host path,
#     # or an emptyDir volume if Scanner V4 is running on a secured cluster without default StorageClass.
#     #
#     # For performance reasons it is HIGHLY RECOMMENDED to use a PVC.
#     # Following values show how to apply custom configuration to a PVC:
```

**Review by johannes94 — CHANGES_REQUESTED** [review]

File: image/templates/helm/stackrox-secured-cluster/values-private-scanner-v4.yaml.example [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# # Private configuration options for Scanner V4 configured as part of the
+# # secured-cluster-services Helm chart.
+# #
+# # NOTE: Scanner V4 cannot be installed on secured clusters using manifest bundles.
```
[the remark this answers is not in the document]
Comment (mclasmeier) [reply image/templates/helm/stackrox-secured-cluster/values-private-scanner-v4.yaml.example:4]: I copied that over from the other example files. 🤷

File: image/templates/helm/stackrox-secured-cluster/values-private-scanner-v4.yaml.example [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+#         <PEM-encoded Scanner V4 indexer service certificate>
+#         -----END CERTIFICATE-----
+#       key: |
+#         -----BEGIN RSA PRIVATE KEY-----
+#         <PEM-encoded Scanner V4 indexer service cert private key>
+#         -----END RSA PRIVATE KEY----
+#
+#  # Configuration for the DB component:
+#  db:
+#     # The password to be used for authenticating database access. This is not user-relevant
```
[the remark this answers is not in the document]
Comment (mclasmeier) [reply image/templates/helm/stackrox-secured-cluster/values-private-scanner-v4.yaml.example:24]: get it. Will change. thanks.

File: image/templates/helm/stackrox-secured-cluster/values-public-scanner-v4.yaml.example [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+#       role: scanner-v4
+#
+#     # If the nodes selected by the node selector are tainted, you can specify the corresponding taint tolerations here.
+#     tolerations:
+#       - effect: NoSchedule
+#         key: infra
+#         value: reserved
+#       - effect: NoExecute
+#         key: infra
+#         value: reserve
```
[the remark this answers is not in the document]
Comment (mclasmeier) [reply image/templates/helm/stackrox-secured-cluster/values-public-scanner-v4.yaml.example:55]: good catch :)

**Review by johannes94 — APPROVED** [review]

**Pull request merged**